package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rollup-plugin-sri",
-  "version": "1.3.4",
+  "version": "1.3.5",
```

Comment:
Can you revert this and add a changefile? This walks you through the process:

https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md

Reply:
Ja kein Problem. Die Changeset Meldung hatte ich gesehen aber in deinem Repsitory sind bisher keine drinn dachte dachte ich die nutzt das Feature gar nicht. In der Beschreibung wird nur auf npx und yarn verwiesen... ich versuche einfach bei npm zu bleiben damit die Komplexität nicht aus dem Ruder läuft.

Reply:
Version 1.4.0 wäre richtig oder? Dafür würde ich dann auch im changelog.md noch einen Eintrag machen. Meintest du das mit changeset?